website/docs/cloud-docs/run/modes-and-options.mdx

```diff
@@ -17,6 +26,44 @@ Terraform Cloud runs support many of the same modes and options available in the
 - **API:** Use the `is-destroy` option.
 - **UI:** Use a workspace's **Destruction and Deletion** settings page.
 
+## Plan Only/Speculative Plan
+
+This option creates a [speculative plan](/terraform/cloud-docs/run/remote-operations#speculative-plans). The speculative plan shows a set of possible changes and checks them against Sentinel policies, but Terraform cannot apply it.
+
+You can create speculative plans with a Terraform version that is different from the one currently selected for the workspace. This lets you check whether your configuration is compatible with a newer Terraform version without changing the workspace settings.
+
+Plan-only runs ignore the per-workspace run queue; they can proceed even if another run is in progress, they cannot become the workspace's current run, and they do not block progress on other runs in the workspace.
+
+- **API:** Set the `plan-only` option to `true` and specify an available terraform version using the `terraform-version` field.
+- **UI:** Open the workspace's **Actions** menu, select **Start new run**, and then choose **Plan only** as the run type.
+- **VCS:** When a workspace is connected to a VCS repository, Terraform Cloud automatically starts a speculative plan when a pull request (or merge request) is opened against the selected branch of that repository. The plan will be linked from the pull/merge request view in your VCS, and can also be found in the workspace's run list.
+
+## Saved Plans
+
+-> **Version note:** Using saved plans from the CLI with Terraform Cloud requires at least Terraform CLI v1.6.0.
+
+Saved plan runs are very similar to standard plan and apply runs: they perform a plan, and then optionally apply it. There are three main differences:
+
+1. _No wait for planning._ Saved plan runs ignore the per-workspace run queue during their plan and checks; like plan-only runs, they can begin planning even if another run is in progress, without blocking progress on other runs. They only become the current run (and block other runs) once they are confirmed for apply.
+2. _No auto-apply._ Saved plan runs are never auto-applied, even if auto-apply is enabled for the workspace; they will only apply when confirmed by a user.
+3. _Automatic discard for stale plans._ If another run is applied (or the state is otherwise modified) before a saved plan run is confirmed, Terraform Cloud will automatically discard that saved plan, since it would no longer make sense to apply it. Saved plans may also be automatically discarded if they are not confirmed within a few weeks.
```

Comment:
Should we tell folks how to automatically discard saved plans? Or does Terraform Cloud do that without folks having to enable it?
```suggestion
3. _Automatic discard for stale plans._ If another run is applied (or the state is otherwise modified) before a saved plan run is confirmed, Terraform Cloud automatically discards that saved plan. You can also automatically discard saved plans if you do not are confirm them within a few weeks.
```

Reply:
That's default behavior that you cannot opt out of.

Reply:
```suggestion
3. _Automatic discard for stale plans._ If another run is applied (or the state is otherwise modified) before a saved plan run is confirmed, Terraform Cloud automatically discards that saved plan. Terraform Cloud may also automatically discard saved plans if they are not confirmed within a few weeks.
```